_field-types/mapping-parameters/index-phrases.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: default
+title: Index phrases
+parent: Mapping parameters
+grand_parent: Mapping and field types
+nav_order: 80
+has_children: false
+has_toc: false
+---
+
+# Index phrases
+
+The `index_phrases` mapping parameter determines whether a field’s text is additionally processed to generate phrase tokens. When enabled, the system creates extra tokens that represent consecutive word sequences. This can significantly improve the performance and accuracy of phrase queries. However, it also increases the index size and the time needed to index documents.
```

Comment:
`consecutive word sequences`

Can we call out that it is just 2 consecutive words? The javadoc here helps understand how tokens are generated.

https://github.com/apache/lucene/blob/ccc8f7e245e76a1c3974770e234fa03d8a905316/lucene/analysis/common/src/java/org/apache/lucene/analysis/shingle/FixedShingleFilter.java#L37C7-L38C98

Reply:
@mgodwan thank you for the review, I've made changes, could you please review?